src/app/services/views/ServiceDetailView.vue

```diff
@@ -22,98 +22,229 @@
-                        name: { description: 'filter by name or parts of a name' },
-                        protocol: { description: 'filter by “kuma.io/protocol” value' },
-                        tag: { description: 'filter by tags (e.g. “tag: version:2”)' },
-                        zone: { description: 'filter by “kuma.io/zone” value' },
-                      }"
-                      @fields-change="(val) => route.update({
+                <template #toolbar>
+                  <KFilterBar
+                    data-testid="gateway-type-filter"
+                    class="data-plane-proxy-filter"
+                    :placeholder="`tag: 'kuma.io/protocol: http'`"
+                    :query="props.query"
+                    :fields="{
+                      name: { description: 'filter by name or parts of a name' },
+                      protocol: { description: 'filter by “kuma.io/protocol” value' },
+                      service: { description: 'filter by “kuma.io/service” value' },
+                      tag: { description: 'filter by tags (e.g. “tag: version:2”)' },
+                      zone: { description: 'filter by “kuma.io/zone” value' },
+                    }"
+                    @fields-change="(val) => {
+                      console.log(val)
+                      route.update({
                         query: val.query,
                         s: val.query.length > 0 ? JSON.stringify(val.fields) : ''
-                      })"
-                    />
-                    <template
-                      v-if="gateways"
-                    >
-                      <KSelect
-                        label="Type"
-                        :overlay-label="true"
-                        :items="[
-                          {
-                            label: 'All',
-                            value: 'all'
-                          },
-                          {
-                            label: 'Builtin',
-                            value: 'builtin'
-                          },
-                          {
-                            label: 'Delegated',
-                            value: 'delegated'
-                          }
-                        ].map(item => ({
-                          ...item,
-                          selected: item.value === props.gatewayType
-                        }))"
-                        appearance="select"
-                        @selected="(item: SelectItem) => route.update({
-                          gatewayType: String(item.value),
-                        })"
+                      })
+                    }"
+                  />
+                  <KSelect
+                    label="Type"
+                    :overlay-label="true"
+                    :items="[
+                      {
+                        label: 'All',
+                        value: 'all'
+                      },
+                      {
+                        label: 'Dataplane Proxy',
```

Comment:
```suggestion
                        label: 'Dataplane Proxies',
```

Reply:
Any reason not to do the i18n now?

Reply:
I could do, but I want to keep an already busy PR as less-busy/change-y as possible.

I was also hoping to do a i18n PR across all list views once the entire piece of "list view" work is done.

So TLDR I'd rather not do it here, but not too precious about it if you would prefer to do it now.